testdata/p4_16_samples/pins/pins_fabric.p4

```diff
@@ -372,7 +372,7 @@ control routing(in headers_t headers, inout local_metadata_t local_metadata, ino
         local_metadata.route_metadata = route_metadata;
     }
     @id(0x0100000F) action trap() {
-        clone(CloneType.I2E, 1024);
```

Comment:
@smolkaj  Just a quick clarification question. Is the session ID value here intentional? I had to change it to 511 because the open-source P4Runtime server [only allows session IDs numbering lower than 512](https://github.com/p4lang/PI/blob/4961fb9fb7a03b8fe7f1511f05fc7a0238b1d88c/proto/frontend/src/pre_clone_mgr.h#L86).

Reply:
Ah great, that makes things easier. Do you know whether or when there is a refresh coming to the files at 
https://github.com/sonic-net/sonic-pins/tree/main/sai_p4/instantiations/google 
?

Reply:
@rhalstea do you know the answer?

Reply:
Generally we would be happy to have our up-to-date programs upstreamed to p4c, we should just keep a copyright notice.

Reply:
We usually only update the programs in https://github.com/sonic-net/sonic-pins/tree/main/sai_p4/instantiations/google when we're upstreaming new features into a SONiC release which have around a 6 month cadence. However, we don't always upstream new P4 features each release.

If we want to have up-to-date programs I like @smolkaj's suggestion of p4c. The copies in sonic-pins need to be kept in sync with the P4RT App so as to not break the component tests, and that dependency would be the bottleneck.

Reply:
I guess I didn't explicitly answer @fruffy's question about the next refresh ... It won't we anytime soon. The feature's we're targeting for the May SONiC release don't require a P4 program updates, and we haven't decided on the features to upstream for the next November release.

Reply:
Thanks, that makes sense. In the meantime we can fix issues manually until the next refresh. Or we could update out-of-band (directly from the private repo to p4c's versions) , if necessary.  Currently, there is no pressing need.